koku/masu/external/kafka_msg_handler.py

```diff
@@ -536,6 +547,44 @@ def summarize_manifest(report_meta):
     return async_id
 
 
+def convert_manifest_to_parquet(report_meta):
+    """
+    Kick off manifest conversion when all report files have completed line item processing.
+
+    Args:
+        report (Dict) - keys: value
+                        schema_name: String,
+                        manifest_id: Integer,
+                        provider_uuid: String,
+                        provider_type: String,
+                        date: DateTime
+
+    Returns:
+        Celery Async UUID.
+
+    """
+    async_id = None
+    schema_name = report_meta.get("schema_name")
+    manifest_id = report_meta.get("manifest_id")
+    provider_uuid = report_meta.get("provider_uuid")
+    schema_name = report_meta.get("schema_name")
+    provider_type = report_meta.get("provider_type")
+    start_date = report_meta.get("date")
+
+    with ReportManifestDBAccessor() as manifest_accesor:
+        manifest = manifest_accesor.get_manifest_by_id(manifest_id)
+        if manifest.num_processed_files == manifest.num_total_files:
```

Comment:
This looks like the same check we do for when to call `summarize_reports.delay`. I'll take it as a TODO for me to consolidate this once I merge the worker changes.  Changes will be needed anyway since I'm moving away from `num_processed_files`